src/library/scala/Function1.scala

```diff
@@ -53,3 +53,11 @@ trait Function1[@specialized(scala.Int, scala.Long, scala.Float, scala.Double) -
 
   override def toString() = "<function1>"
 }
+
+object Function1 {
```

Comment:
Note that this file is generated, see https://github.com/scala/scala/blob/2.13.x/src/build/genprod.scala

Reply:
The implicit class is moved to `Predef` now